task/rpm-ostree/0.1/README.md

```diff
@@ -0,0 +1,31 @@
+# rpm-ostree task
```

Comment:
Is there a good reason to have this in the same commit?

Reply:
I regenerated all README.md's for the Tasks I changed. The better question might be why all the other authors did not bother to do that.

Reply:
Ack, it makes sense, but consider explaining that in the commit message, or putting those changes in another commit.

Reply:
I have, it stays in the commit and the message remains the same.